frontend/src/metabase/nav/components/AdminNavbar/utils.tsx

```diff
@@ -0,0 +1,50 @@
+import _ from "underscore";
+
+import { measureNormalTextWidth } from "metabase/lib/measure-text";
+import type { FontStyle } from "metabase/visualizations/shared/types/measure-text";
+import type { AdminPath } from "metabase-types/store";
+
+const linkStyle: Partial<FontStyle> = {
+  size: "13.6px",
+};
+const boldTextStyle: Partial<FontStyle> = {
+  ...linkStyle,
+  weight: "bold",
+};
+
+const defaultBreakpoint = 1280;
+
+/** The breakpoint is based on the width of the items in the nav */
+export const getAdminNavBreakpoint = (
+  adminPaths: AdminPath[],
+  adminLogoText: string,
+  exitButtonText: string,
+) => {
+  try {
+    const navPadding = 32;
+    const iconWidth = 24;
```

Comment:
This is a lot of magic numbers used to calculate the breakpoint. I like the Idea of the breakpoint for this menu being more dynamic (especially since language can have an impact on the size). Rather than measure the text like this, could we do something similar to what we do for table visualizations and render offscreen to measure? That should give us an accurate idea of what the breakpoint actually is and capture any future style changes.